api/src/main/resources/messages.properties

```diff
@@ -129,7 +129,7 @@ mirebalais.vitals.ratioMuacHC.title=(Calculated) MUAC/HC
 mirebalais.vitals.ratioMuacHC.instructions= If Mid-Upper Arm Circumference (MUAC) and Head circumference (HC) are recorded, the ratio (MUAC/HC) will be calculated.
 mirebalais.vitals.ratioMuacHC.display=Patient's MUAC/HC ratio is:
 mirebalais.vitals.zscore.title=Zscore
-mirebalais.vitals.response.title=Level of consciousness
+mirebalais.vitals.LevelOfConsciousness.title=Level of consciousness
```

Comment:
The normative casing here is `camelCase`, i.e., initial letter small. But we'll call this good for now.